cli-helpers/src/commands/migrate-avl-subscriptions.ts

```diff
@@ -0,0 +1,142 @@
+                {
+                    name: "fileName",
+                    message: "Enter the file name for the subscriptions CSV file",
+                    type: "input",
+                },
+            ]);
+
+            fileName = responses.fileName;
+        }
+
+        const subscriptionsJson = csvToJson
+            .fieldDelimiter(",")
+            .getJsonFromCsv(`${fileName}.csv`) as unknown as BodsSubscription[];
+
+        const setOfSubscriptionUrlAndUsername = new Set();
+        const dataProducersToSubscribeTo: Subscription[] = [];
+
+        subscriptionsJson.map((subscription) => {
+            if (subscription.status === "live") {
+                setOfSubscriptionUrlAndUsername.add(`${subscription.url_link}, ${subscription.username}`);
+
+                dataProducersToSubscribeTo.push({
+                    id: subscription.dataset_id,
+                    publisherId: subscription.organisation_id,
+                    url: subscription.url_link,
+                    username: subscription.username,
+                    password: subscription.password,
+                });
+            }
+        });
+
+        subscriptionsJson.map((subscription) => {
+            if (
+                !setOfSubscriptionUrlAndUsername.has(`${subscription.url_link}, ${subscription.username}`) &&
+                subscription.status === "error"
+            ) {
+                setOfSubscriptionUrlAndUsername.add(`${subscription.url_link}, ${subscription.username}`);
+
+                dataProducersToSubscribeTo.push({
+                    id: subscription.dataset_id,
+                    publisherId: subscription.organisation_id,
+                    url: subscription.url_link,
+                    username: subscription.username,
+                    password: subscription.password,
+                });
+            }
+        });
+
+        const unsuccessfulSubscriptions: Subscription[] = [];
+        const successfulSubscriptions: Subscription[] = [];
+
+        for (const subscription of dataProducersToSubscribeTo) {
+            const payload = generateLambdaPayload(
+                subscription.url,
+                subscription.username,
+                subscription.password,
+                subscription.id,
+                subscription.publisherId,
+            );
+
+            const subscribeEvent = await invokeLambda(stage, {
+                FunctionName: `integrated-data-avl-subscriber-${stage}`,
+                InvocationType: "RequestResponse",
+                Payload: JSON.stringify(payload),
+            });
+
+            if (!subscribeEvent || !subscribeEvent.Payload) {
+                unsuccessfulSubscriptions.push(subscription);
+                return;
+            }
+
+            const returnPayload: Uint8ArrayBlobAdapter = subscribeEvent.Payload;
+
+            const returnPayloadJson = JSON.parse(returnPayload?.transformToString() ?? "{}");
```

Comment:
```suggestion
            const returnPayloadJson = JSON.parse(subscribeEvent.Payload?.transformToString() ?? "{}");
```

Doing it inline like this means we can avoid installing and using the `@smithy/util-stream` package

Same for the rollback script